ide/editor.lib2/src/org/netbeans/api/editor/EditorActionRegistrations.java

```diff
@@ -30,8 +30,8 @@
  * <br>
  * Example:
  * <pre>
- * @EditorActionRegistrations({ @EditorActionRegistration(name = "name1", ...),
- *                              @EditorActionRegistration(name = "name2", ...)
+ * &#64;EditorActionRegistrations({ &#64;EditorActionRegistration(name = "name1", ...),
+ *                              &#64;EditorActionRegistration(name = "name2", ...)
  *                            })
```

Comment:
nit: maybe a candidate for a `@code` section

Reply:
even with @code it still needs to be escaped or it does'nt know @EditorActionRegistration command :D